examples/basic/empiricalrc.json

```diff
@@ -18,6 +18,7 @@
       "model": "gpt-4-turbo-preview",
       "prompt": "Extract the name, age and location from the message, and respond with a JSON object. If an entity is missing, respond with null.\n\nMessage: {{user_message}}",
       "parameters": {
+        "timeout": 3000,
```

Comment:
let's not add it here. good to minimize the amount of info that a user needs to go through